app/helpers/utils.js

```diff
@@ -0,0 +1,21 @@
+const bcrypt = require('bcryptjs');
+const config = require('../../config');
+
+exports.encryptar = password =>
+  new Promise((resolve, reject) => {
+    bcrypt.genSalt(parseInt(config.common.roundsBcrypt), (errGenSalt, salt) => {
+      if (errGenSalt) reject(errGenSalt);
+      bcrypt.hash(password, salt, (errHash, hash) => {
+        if (errHash) reject(errHash);
+        resolve(hash);
+      });
+    });
+  });
+
+exports.comparePassword = (password, hashPassword) =>
```

Comment:
Remember to follow the card requirements this is fine, but for the moment it is not necessary for the card because it is not yet the signIn part. 

also for this the library has a synchronous method compareSync